packages/registration_delivery/lib/models/entities/registration_delivery_enums.dart

```diff
@@ -9,4 +9,6 @@ enum RegistrationDeliveryEnums {
   name,
   @MappableValue("userName")
   userName,
+  @MappableValue("IEC")
```

Comment:
This we be moved to checklist enum

Reply:
We will move it to checklist type enums if more checklist checks comes up